app/controllers/state_file/questions/nj_homeowner_eligibility_controller.rb

```diff
@@ -3,6 +3,45 @@ module Questions
     class NjHomeownerEligibilityController < QuestionsController
       include ReturnToReviewConcern
 
+      before_action -> { set_checkboxes }
+
+      def set_checkboxes
+        @checkbox_collection = [
+          {
+            method: :homeowner_home_subject_to_property_taxes,
+            label: I18n.t("state_file.questions.nj_homeowner_eligibility.edit.homeowner_home_subject_to_property_taxes")
+          },
+          { 
+            method: :homeowner_more_than_one_main_home_in_nj,
+            label: I18n.t("state_file.questions.nj_homeowner_eligibility.edit.homeowner_more_than_one_main_home_in_nj"),
+          },
+          { 
+            method: :homeowner_shared_ownership_not_spouse,
+            label: I18n.t("state_file.questions.nj_homeowner_eligibility.edit.homeowner_shared_ownership_not_spouse"),
+          },
+          {
+            method: :homeowner_main_home_multi_unit,
+            label: I18n.t("state_file.questions.nj_homeowner_eligibility.edit.homeowner_main_home_multi_unit"),
+            has_follow_up_id: 'homeowner_multi_unit_followup'
+          },
+          {
+            method: :homeowner_main_home_multi_unit_max_four_one_commercial,
+            label: I18n.t("state_file.questions.nj_homeowner_eligibility.edit.homeowner_main_home_multi_unit_max_four_one_commercial"),
+            is_follow_up_id: 'homeowner_multi_unit_followup'
```

Comment:
[dust] - i was initially confused by the difference between `has_follow_up_id` and `is_follow_up_id`... to me, when something *has* an ID of `12345` that also means it *is* the thing with that ID.

Thoughts about wording like this?

`has_follow_up_with_id`
`follow_up_id`

Reply:
Yeah that seems better! Thanks for the suggestions!